### PR TITLE
Feature 50 summary page

### LINKED ISSUE #50: Summary page
As a user I would like to view the survey summary

### PR DESCRIPTION
**Pull request by Chi-Chu319**
resolves #50

### CODE REVIEW

File: src/components/survey/summary.tsx [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+                      flexItem
+                    />
+                    { surveyReusable.amountAsWaste && renderMediaDataCell(strings.survey.reusables.dataGridColumns.wasteAmount, materialAmountAsWaste) }
+                  </Stack>
+                  { renderMediaDataCell(strings.survey.reusables.dataGridColumns.description, surveyReusable.description) }
+                </Stack>
+              </Paper>
+            );
+          })
+          }
```
Comment (lauriLukkarinen) [inline src/components/survey/summary.tsx:561]: Extract inline function to its own method

File: src/components/survey/summary.tsx [hunk 2]
```diff
@@ -0,0 +1,719 @@
...
+      return null;
+    }
+
+    return (
+      <Stack spacing={ 2 }>
+        <Typography variant="h3">
+          { strings.survey.wasteMaterial.title }
+        </Typography>
+        <Stack spacing={ 2 }>
+          { wastes.map(waste => {
```
Comment (lauriLukkarinen) [inline src/components/survey/summary.tsx:581]: extract

File: src/components/survey/summary.tsx [hunk 3]
```diff
@@ -0,0 +1,719 @@
...
+      return null;
+    }
+
+    return (
+      <Stack spacing={ 2 }>
+        <Typography variant="h3">
+          { strings.survey.hazardousMaterial.title }
+        </Typography>
+        <Stack spacing={ 2 }>
+          { hazardousWastes.map(hazardousWaste => {
```
Comment (lauriLukkarinen) [inline src/components/survey/summary.tsx:640]: extract

File: src/localization/en.json [hunk 4]
```diff
@@ -12,6 +12,10 @@
     "open": "Open"
   },
 
+  "units": {
+    "tons": "tonnia"
```
Comment (lauriLukkarinen) [inline src/localization/en.json:16]:
```suggestion
    "tons": "tons"
```

File: src/localization/fi.json [hunk 5]
```diff
@@ -12,6 +12,10 @@
     "open": "Avaa"
   },
 
+  "units": {
+    "tons": "tonnia"
+  },
+
   "errorHandling": {
     "title": "Error",
```
Comment (lauriLukkarinen) [inline src/localization/fi.json:20]:
```suggestion
    "title": "Virhe",
```

**Review by lauriLukkarinen — CHANGES_REQUESTED** [review]

**Review by Chi-Chu319 — COMMENTED** [review]
comment

**Review by lauriLukkarinen — APPROVED** [review]

**Pull request merged**